workspace/exercise.go

```diff
@@ -37,7 +37,12 @@ func (e Exercise) Filepath() string {
 
 // MetadataFilepath is the absolute path to the exercise metadata.
 func (e Exercise) MetadataFilepath() string {
-	return filepath.Join(e.Filepath(), solutionFilename)
+	return filepath.Join(e.Filepath(), ignoreSubdirMetadataFilepath())
```

Comment:
How would you feel about making the relative path to the metadata filepath a variable defined at compile time rather than a function call?

```
var metadataFilepath = filepath.Join(".exercism", "solution.json")
```

Reply:
Sure. I'd made it a function call as a workaround to keep `const`.

Reply:
Ah right. Personally, I'd rather have a var than a function call